solidity/test/random_beacon_operator/TestSlashing.js

```diff
@@ -19,6 +23,13 @@ describe('KeepRandomBeaconOperator/Slashing', function () {
     anotherOperatorContract = accounts[6],
     registryKeeper = accounts[7];
 
+  let scheduleStart
+  let relayRequestStartBlock
+
+  const largeStake = web3.utils.toBN("50000000000000000000000000") // 50 000 000 KEEP
+  const mediumStake = web3.utils.toBN("500000000000000000000000") // 500 000 KEEP
```

Comment:
I think it would be nice to align the numbers with 100%, 50% and 1%, so 50 000 000K, 25 000 000K and 50 000. Up to you :)